architecture/understanding-development.adoc

```diff
@@ -1,45 +1,396 @@
+registries, and running them, see
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/managing_containers/index%23using_podman_to_work_with_containers[Using podman to work with containers]
+and link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/managing_containers/index%23building_container_images_with_buildah[Building container images with buildah].
+
+[id="choosing-container-build-tools"]
+=== Choosing container build tools
+
+While the Docker Container Engine and `docker` command are popular tools
+to work with containers, with RHEL and many other Linux systems, you can
+instead choose a different set of container tools that includes podman, skopeo,
+and buildah. You can still use Docker Container Engine tools to create
+containers that will run in {product-title} and any other container platform.
+
+Building and managing containers with buildah, podman, and skopeo results in
+industry standard container images that include features tuned specifically
+for ultimately deploying those containers in {product-title} or other Kubernetes
+environments. These tools are daemonless and can be run without root privileges,
+so there is less overhead in running them.
+
+When you ultimately run your containers in {product-title}, you use the
+link:https://cri-o.io/[CRI-O] container engine. CRI-O runs on every worker and
+master machine in an {product-title} cluster, but CRI-O is not yet supported as
+a standalone runtime outside of {product-title}.
+
+[id="choosing-base-image"]
+=== Choosing a base image
+
+The base image you choose to build your application on contains a set of
+software that resembles a Linux system to your application. When you build your
+own image, your software is placed into that file system and sees that file
+system as though it were looking at its operating system. Choosing this base
+image has major impact on how secure, efficient and upgradeable your container
+is in the future.
+
+Red Hat provides a new set of base images referred to as
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/getting_started_with_containers/index%23using_red_hat_base_container_images_standard_and_minimal[Red Hat Universal Base Images] (UBI).
+These images are based on Red Hat Enterprise Linux and are similar to base
+images that Red Hat has offered in the past, with one major difference: they
+are freely redistributable without a Red Hat subscription. As a result, you can
+build your application on UBI images without having to worry about how they
+are shared or the need to create different images for different environments.
+
+These UBI images have standard, init, and minimal versions. You can also use the
+link:https://access.redhat.com/documentation/en-us/red_hat_software_collections/3/html-single/using_red_hat_software_collections_container_images/index[Red Hat Software Collections]
```

Comment:
This is probably fine for now, since some of our images content on using images hasn't yet been brought forward.

Reply:
If you have a better link after you bring the images content from 3.x forward, will you swap it out?